okhttp-android-support/src/test/java/okhttp3/internal/huc/CacheAdapterTest.java

```diff
@@ -116,7 +116,7 @@
     };
     setInternalCache(new CacheAdapter(responseCache));
     client = client.newBuilder()
-        .sslSocketFactory(sslContext.getSocketFactory())
+        .sslSocketFactory(sslContextBuilder.socketFactory(), sslContextBuilder.trustManager())
```

Comment:
Two different things.

1. Split SSLContextBuilder into two classes: SSLClient and SSLClient.Builder.
2. Move the new methods in DefaultSecurity to Util. (Though I don’t feel strongly here.)